live-examples/js-examples/array-foreach.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">const array1 = ['a', 'b', 'c'];
```

Comment:
We should declare this with `var` not `const` (see https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#language-choice-es5es6).